### PR TITLE
Getting rid of bad monitoring script and Changing type of wait signal

### PR DESCRIPTION
**Pull request by goru97**
*What:
1. Getting rid of bad monitoring script.
2. Changing type of wait condition from OS::Heat::SwiftSignal to AWS::CloudFormation::WaitCondition

*Why:
1. Monitoring script was failing somewhere in-between making wc_notify condition unreachable and causing error after wait condition timed out.
2. People occasionally saw missed signals with OS::Heat::SwiftSignal; AWS::CloudFormation::WaitCondition is better resource in comparison.

### CODE REVIEW

File: grafana-cloud-metrics.yaml [hunk 1]
```diff
@@ -118,14 +118,14 @@ outputs:
 
 resources:
   wait_condition:
-    type: "OS::Heat::SwiftSignal"
+    type: "AWS::CloudFormation::WaitCondition"
```
Comment (rs-randallburt) [inline grafana-cloud-metrics.yaml]: These are actually the same resource under the hood. AWS::CloudFormation::WaitCondition is an alias for compatibility purposes.

**Comment by rs-randallburt** [comment]
:-1:

File: grafana-cloud-metrics.yaml (hunk 1 above)
Comment (goru97) [inline grafana-cloud-metrics.yaml]: @miguelgrinberg suggested this change!

Comment (miguelgrinberg) [inline grafana-cloud-metrics.yaml]: @rs-randallburt I did not know this. We had some issues with signals that didn't fire when using `SwiftSignal`, the AWS wait condition appeared to be more stable.

**Comment by goru97** [comment]
@rs-randallburt Reverted that change!

**Pull request merged**